src/main/java/nextstep/qna/domain/Question.java

```diff
@@ -72,19 +73,43 @@ public boolean isOwner(NsUser loginUser) {
         return writer.equals(loginUser);
     }
 
-    public Question setDeleted(boolean deleted) {
-        this.deleted = deleted;
-        return this;
+    public void updateDeleted() {
+        this.deleted = true;
     }
```

Comment:
```suggestion
    private void delete() {
        this.deleted = true;
}
```
외부에서 삭제 상태를 바꿀 수 없도록 위와 같이 구현하는 것은 어떨까?